### DISCUSSION TITLE
Regarding rename of the primary branch

### DISCUSSION THREAD

**Discussion by EtiamNullam** [noib3/cokeline.nvim#116] [General]
Was the primary branch changed recently to `main` from `master`? I can adjust but seems like `Lazy` (a package manager) still tries to reach old branch and fails:

```
fatal: couldn't find remote ref refs/heads/master
```

I don't see a point in renaming the primary branch unless it's necessary as it can break things for many, even if it's just a small nuisance.

For anyone that is affected: simply uninstall and install the plugin again.

**Comment by willothy** [comment]
I renamed just to follow general convention. However, I didn't expect breakage from package managers unless people had the master branch pinned. Do you think it would be worth restoring the master branch and mirroring changes from main to give people time to switch over, or is this post enough?

**Reply by EtiamNullam** [reply]
I think it depends when did the rename happen and how many people already had to deal with this issue. If it's very recently I would simply revert to `master`. Otherwise it might be worth it to keep both for a while.

I think staying with `master` is favorable as that's the standard, while `main` sounds more like a fashion.

**Reply by willothy** [reply]
As far as I know main is the new standard, but I guess that's still up for debate. I would prefer to keep the `main` label, but I will mirror `main` to a new `master` branch so that lazy can resolve `refs/heads/master`.
